src/services/add-query-to-queue.ts

```diff
@@ -118,6 +128,10 @@ export default class AddQueryToQueue {
       newSongs = shuffle(newSongs);
     }
 
+    if (this.config.ENABLE_SPONSORBLOCK && !this.sbClientTimeout) {
```

Comment:
let's have the `!this.sbClientTimeout` check in the added function

Reply:
The `enrichWithSkipSegments` method already check for it
```js
    if (!this.sponsorBlock
          || this.sponsorBlockTimeout
          || song.source !== MediaSource.Youtube
          || !song.url)
         ``
I added a slight redundancy to skip the `.map` entirely, but if you think it's making the  code harder to read I can remove it !

Reply:
looks good now 👍